projects/notebooks/visualizations/rhods-notebooks-performance/models/lts.py

```diff
@@ -37,15 +37,16 @@ class BenchmarkMeasures(matbench_models.ExclusiveModel):
 
 class Results(matbench_models.ExclusiveModel):
     benchmark_measures: BenchmarkMeasures
+    regression: NotebookPerformanceRegression
 
 NotebookPerformanceKPIs = matbench_models.getKPIsModel("NotebookPerformanceKPIs", __name__, kpi.KPIs, kpi.NotebookPerformanceKPI)
+NotebookPerformanceRegression = matbench_models.RegressionResult
```

Comment:
this isn't necessary, if you don't modify `matbench_models.RegressionResult`, no need to rename it here